Automattic/jetpack
# PHPCS: Like module

[(Not to be confused with **Comment** Likes Module.)](https://github.com/Automattic/jetpack/pull/19221)

This PR updates the likes module to meet phpcs standards.

#### Changes proposed in this Pull Request:

* This PR updates the likes module to meet phpcs standards.

#### Jetpack product discussion



N/a

#### Does this pull request change what data or activity we track or use?



Nope.

#### Testing instructions:






* Check out branch
* Make sure you can still enable likes under Settings > Sharing.
* Make sure you can still like posts and pages without an issue, manage email notification settings, etc.
